Quaver.Shared/QuaverGame.cs

```diff
@@ -947,7 +947,6 @@ public static void SetAudioDevice(bool reloadResources = false)
             {
                 if (ConfigManager.AudioOutputDevice.Value != Bass.GetDeviceInfo(i).Name)
                     continue;
-
```

Comment:
Whats with the random whitespace removal?

Reply:
Fixed